compiler/testData/codegen/bytecodeListing/jvmStatic/jvmStaticExternalWithDeclarationRewrite.kt

```diff
@@ -0,0 +1,15 @@
+// WITH_STDLIB
```

Comment:
Please, use `// TARGET_BACKEND: JVM_IR`, otherwise, the test will run on other backends, including Old JVM, which is not the focus of the test. Additionally, you will get rid of `txt` files for old backend.

Reply:
Again, this wasn't something in the tests these are copied from. Apologies for that. I'll add this.